src/components/common/Questionnaire/CollapsedDrawer/index.js

```diff
@@ -27,13 +34,16 @@ const CollapsedDrawer = ({ title = '給我們回饋', children }) => {
   }
 
   const childrenWithProps = React.Children.map(children, child =>
-    React.cloneElement(child, { handleToggleModalOpen }),
+    React.cloneElement(child, { handleToggleModalOpen, isExpanded }),
   );
 
-  if (isExpanded) return <Fragment>{childrenWithProps}</Fragment>;
+  if (isExpanded) return childrenWithProps;
 
   return (
-    <div className={styles.container} onClick={handleToggleModalOpen}>
+    <div
+      className={cn(styles.container, { [styles.applyStyle]: isApplyStyle })}
```

Comment:
看起來 `isApplyStyle` 只會發生在 `!isExpanded` 所以這裡的 `isApplyStyle` 應該總是 `true`，所以應該是以下這樣？

```suggestion
      className={cn(styles.container, styles.applyStyle)}
```

Reply:
甚至，要不要直接合併 `.applyStyle` 到 `.container`

Reply:
喔！我知道了，一開始會這樣寫，是因為一直無法修正位移的問題，以為是要等到 useEffect 的時候修改 css 才會正確，所以才會用 applyStyle。

![image](https://github.com/goodjoblife/GoodJobShare/assets/30387546/e0ff775a-0faa-4d62-b397-56f399e653cc)

剛剛試著合併，原來是 display 的問題才對，最後不用 applyStyle。
![image](https://github.com/goodjoblife/GoodJobShare/assets/30387546/2889763a-82e7-43df-9b55-62597a547cc5)

謝謝指出～